l10n/en/firefox/browsers/compare/chrome.ftl

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/compare/chrome/
+
+# Page title
+compare-chrome-firefox-vs-google-chrome = { -brand-name-firefox } vs Google Chrome
+
+# HTML page description
+compare-chome-read-our-comparison = Read our comparison of { -brand-name-firefox } and Chrome browsers on features, privacy, and ease-of-use.
+
+# Variables:
+# $fingerprint (url) - link to https://www.mozilla.org/firefox/features/block-fingerprinting/
+# $adblocker (url) - link to https://www.mozilla.org/firefox/features/adblocker/
+compare-chrome-firefoxs-default-privacy-settings = { -brand-name-firefox }’s default privacy settings are much stronger than Chrome’s – and we have additional features to prevent websites from tracking you, such as <a { $fingerprint }>fingerprint blocking</a>, as well as your choice of Add-ons for <a { $adblocker }>Ad Blocking</a>.
```

Comment:
**Nit:** i think these should be lowercase
```suggestion
compare-chrome-firefoxs-default-privacy-settings = { -brand-name-firefox }’s default privacy settings are much stronger than Chrome’s – and we have additional features to prevent websites from tracking you, such as <a { $fingerprint }>fingerprint blocking</a>, as well as your choice of add-ons for <a { $adblocker }>ad blocking</a>.
```